client/a8c-for-agencies/sections/marketplace/hosting-overview/hosting-v2/premier-agency-hosting/index.tsx

```diff
@@ -36,18 +41,29 @@ export default function PremierAgencyHosting( { onAddToCart }: Props ) {
 	const isPressableReferralsEnabled = isEnabled( 'a4a-pressable-referrals' );
 	const showReferralMode = isPressableReferralsEnabled && isReferMode;
 
-	const { pressablePlans } = useProductAndPlans( {
-		selectedSite: null,
-		productSearchQuery: '',
-	} );
+	const { data } = useFetchLicenses(
```

Comment:
A small note that if at some point we will have > 100 referrals we might end up not getting a proper license in the list. It's unlikely for now, but, as discussed we might want to implement a separate endpoint to retrieve the proper pressable plan from the backend.

Reply:
I agree